### PR TITLE
Add certain mediaquerylist and mediquerylistevent translated entries

### PR DESCRIPTION
**Pull request by jacinyan**
1. 添加MediaQueryList条目下的media属性和change事件对象相关翻译
2. 添加MediaQueryListEvent总条目，以及MediaQueryList() 构造函数条目的相关翻译

### CODE REVIEW

**Comment by yin1999** [comment]
Hi @jacinyan, we are now using markdown as document format. Could you rewrite those files, see: <https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md> and <https://developer.mozilla.org/zh-CN/docs/MDN/Contribute/Markdown_in_MDN>.

**Review by yin1999 — CHANGES_REQUESTED** [review]

**Comment by jacinyan** [comment]
> Hi @jacinyan, we are now using markdown as document format. Could you rewrite those files, see: https://github.com/mdn/translated-content/blob/main/docs/zh-cn/translation-guide.md and https://developer.mozilla.org/zh-CN/docs/MDN/Contribute/Markdown_in_MDN.

Sure, will do. BTW, I was wondering if potential amends to any existing pages will also come in markdown from here on out. Cheers

**Comment by jacinyan** [comment]
> > I was wondering if potential amends to any existing pages will also come in markdown from here on out
> 
> Such as:
> 
> * https://developer.mozilla.org/zh-CN/docs/MDN/Contribute/Markdown_in_MDN
> * https://developer.mozilla.org/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Function/apply
> * https://developer.mozilla.org/zh-CN/docs/Web/API/GlobalEventHandlers/onload
> 
> Note that, we **translate** from `en-US`, see the `en-US` document by:
> 
> image

Thanks for letting me know about that @yin1999.

Yet I think you might've misunderstood my question haha. Actually, I was asking if you are discarding HTML and switching to markdown, i.e. not only for newly created translation work, but also planning to migrate the previous pages too. Thanks~~~

File: files/zh-cn/web/api/mediaquerylist/change_event/index.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+page-type: web-api-event
+tags:
+  - API
+  - CSSOM View
+  - Event Handler
+  - MediaQueryList
+  - Event
+  - Reference
+  - onchange
+browser-compat: api.MediaQueryList.change_event
```
Comment (yin1999) [inline files/zh-cn/web/api/mediaquerylist/change_event/index.md:13]: This is used by `{{Specifications}}` and `{{Compat}}` macros, and could fall back to `en-US` documents. So we can just remove it.

```suggestion
```

File: files/zh-cn/web/api/mediaquerylist/change_event/index.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  - : 一个字符串，代表着一个序列化后的媒体查询。
+
+## 示例
+
+```js
+var mql = window.matchMedia('(max-width: 600px)');
+
+mql.onchange = (e) => {
+    if (e.matches) {
+    /* 视口等于或小于600像素 */
```
Comment (yin1999) [inline files/zh-cn/web/api/mediaquerylist/change_event/index.md:52]:
```suggestion
    /* 视口等于或小于 600 像素 */
```

File: files/zh-cn/web/api/mediaquerylist/change_event/index.md [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+    console.log('This is a narrow screen — less than 600px wide.')
+  } else {
+    /* 视口大于600像素 */
```
Comment (yin1999) [inline files/zh-cn/web/api/mediaquerylist/change_event/index.md:55]:
```suggestion
    /* 视口大于 600 像素 */
```

File: files/zh-cn/web/api/mediaquerylist/change_event/index.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- [Media queries](/zh-CN/docs/Web/CSS/Media_Queries/Using_media_queries)
+- [Using media queries from code](/zh-CN/docs/Web/CSS/Media_Queries/Testing_media_queries)
```
Comment (yin1999) [inline files/zh-cn/web/api/mediaquerylist/change_event/index.md:72]:
```suggestion
- [媒体查询](/zh-CN/docs/Web/CSS/Media_Queries/Using_media_queries)
- [在代码中使用媒体查询](/zh-CN/docs/Web/CSS/Media_Queries/Testing_media_queries)
```

File: files/zh-cn/web/api/mediaquerylist/matches/index.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+  - Adaptive Design
+  - CSSOM
+  - CSSOM View
+  - DOM
+  - Media Queries
+  - MediaQueryList
+  - Property
+  - Reference
+  - matches
+browser-compat: api.MediaQueryList.matches
```
Comment (yin1999) [inline files/zh-cn/web/api/mediaquerylist/matches/index.md:16]:
```suggestion
```

File: files/zh-cn/web/api/mediaquerylist/matches/index.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- [Media queries](/zh-CN/docs/Web/CSS/Media_Queries/Using_media_queries)
+- [Using media queries from code](/zh-CN/docs/Web/CSS/Media_Queries/Testing_media_queries)
```
Comment (yin1999) [inline files/zh-cn/web/api/mediaquerylist/matches/index.md:63]:
```suggestion
- [媒体查询](/zh-CN/docs/Web/CSS/Media_Queries/Using_media_queries)
- [在代码中使用媒体查询](/zh-CN/docs/Web/CSS/Media_Queries/Testing_media_queries)
```

File: files/zh-cn/web/api/mediaquerylistevent/index.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+slug: Web/API/MediaQueryListEvent
+page-type: web-api-interface
+tags:
+  - API
+  - CSSOM View
+  - Interface
+  - Media Queries
+  - MediaQueryListEvent
+  - Reference
+browser-compat: api.MediaQueryListEvent
```
Comment (yin1999) [inline files/zh-cn/web/api/mediaquerylistevent/index.md:12]:
```suggestion
```

File: files/zh-cn/web/api/mediaquerylistevent/mediaquerylistevent/index.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+  - CSSOM View
+  - Constructor
+  - Media Queries
+  - MediaQueryListEvent
+  - Reference
+browser-compat: api.MediaQueryListEvent.MediaQueryListEvent
+---
+{{APIRef("CSSOM")}}
+
+**`MediaQueryListEvent()`** 创建一个新的 {{DOMxRef("MediaQueryListEvent")}} 对象。
```
Comment (yin1999) [inline files/zh-cn/web/api/mediaquerylistevent/mediaquerylistevent/index.md:16]:
```suggestion
**`MediaQueryListEvent()`** 构造函数用于创建一个新的 {{DOMxRef("MediaQueryListEvent")}} 对象。
```

File: files/zh-cn/web/api/mediaquerylistevent/mediaquerylistevent/index.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+slug: Web/API/MediaQueryListEvent/MediaQueryListEvent
+page-type: web-api-constructor
+tags:
+  - API
+  - CSSOM View
+  - Constructor
+  - Media Queries
+  - MediaQueryListEvent
+  - Reference
+browser-compat: api.MediaQueryListEvent.MediaQueryListEvent
```
Comment (yin1999) [inline files/zh-cn/web/api/mediaquerylistevent/mediaquerylistevent/index.md:12]:
```suggestion
```

File: files/zh-cn/web/api/mediaquerylistevent/index.md [hunk 10]
```diff
@@ -0,0 +1,72 @@
...
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- [Media queries](/zh-CN/docs/Web/CSS/Media_Queries/Using_media_queries)
+- [Using media queries from code](/zh-CN/docs/Web/CSS/Media_Queries/Testing_media_queries)
```
Comment (yin1999) [inline files/zh-cn/web/api/mediaquerylistevent/index.md:70]:
```suggestion
- [媒体查询](/zh-CN/docs/Web/CSS/Media_Queries/Using_media_queries)
- [在代码中使用媒体查询](/zh-CN/docs/Web/CSS/Media_Queries/Testing_media_queries)
```

File: files/zh-cn/web/api/mediaquerylistevent/mediaquerylistevent/index.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+{{Specifications}}
+
+## 浏览器兼容性
+
+{{Compat}}
+
+## 参见
+
+- [Media queries](/zh-CN/docs/Web/CSS/Media_Queries/Using_media_queries)
+- [Using media queries from code](/zh-CN/docs/Web/CSS/Media_Queries/Testing_media_queries)
```
Comment (yin1999) [inline files/zh-cn/web/api/mediaquerylistevent/mediaquerylistevent/index.md:60]:
```suggestion
- [媒体查询](/zh-CN/docs/Web/CSS/Media_Queries/Using_media_queries)
- [在代码中使用媒体查询](/zh-CN/docs/Web/CSS/Media_Queries/Testing_media_queries)
```

File: files/zh-cn/web/api/mediaquerylistevent/index.md [hunk 12]
```diff
@@ -0,0 +1,72 @@
...
+  - CSSOM View
+  - Interface
+  - Media Queries
+  - MediaQueryListEvent
+  - Reference
+browser-compat: api.MediaQueryListEvent
+---
+{{APIRef("CSSOM")}}
+
+`MediaQueryListEvent`对象存储着发生在 {{DOMxRef("MediaQueryList")}} 对象上的变化的信息——作为一个在函数中被{{DOMxRef("MediaQueryList.change_event", "change")}} 事件引用的事件对象，该对象的实例是可用的。
```
Comment (yin1999) [inline files/zh-cn/web/api/mediaquerylistevent/index.md:16]:
```suggestion
`MediaQueryListEvent` 对象存储着发生在 {{DOMxRef("MediaQueryList")}} 对象上的变化的信息——作为一个在函数中被 {{DOMxRef("MediaQueryList.change_event", "change")}} 事件引用的事件对象。
```

File: files/zh-cn/web/api/mediaquerylistevent/index.md [hunk 13]
```diff
@@ -0,0 +1,72 @@
...
+_`MediaQueryListEvent` 接口的方法继承自它的父接口，{{DOMxRef("Event")}}。_
+
+## 示例
+
+```js
+var mql = window.matchMedia('(max-width: 600px)');
+
+function screenTest(e) {
+  if (e.matches) {
+    /* 视口等于或小于600像素 */
```
Comment (yin1999) [inline files/zh-cn/web/api/mediaquerylistevent/index.md:46]:
```suggestion
    /* 视口等于或小于 600 像素 */
```

File: files/zh-cn/web/api/mediaquerylistevent/index.md [hunk 14]
```diff
@@ -0,0 +1,72 @@
...
+    para.textContent = 'This is a narrow screen — less than 600px wide.';
+    document.body.style.backgroundColor = 'red';
+  } else {
+    /* 视口大于600像素 */
```
Comment (yin1999) [inline files/zh-cn/web/api/mediaquerylistevent/index.md:50]:
```suggestion
    /* 视口大于 600 像素 */
```

**Review by yin1999 — APPROVED** [review]
Thanks a lot. Welcome aboard~

**Pull request merged**